pkg/minishift/network/settings_windows.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"time"
+
+	hvkvp "github.com/gbraad/go-hvkvp"
+	"github.com/golang/glog"
+	"github.com/minishift/minishift/pkg/minishift/shell/powershell"
+)
+
+const (
+	resendInterval = time.Second
+)
+
+func ConfigureNetworking(machineName string, networkSettings NetworkSettings) {
+	PrintNetworkSetting(networkSettings)
+
+	networkScript := FillNetworkSettingsScript(networkSettings)
+	record := hvkvp.NewMachineKVPRecord(machineName,
+		"MINISHIFT_NETWORKING",
+		// to allow sending multiple lines in the value we encode the script
+		base64.StdEncoding.EncodeToString([]byte(networkScript)))
+
+	command := hvkvp.NewMachineKVPCommand(record)
+	b := newConfigBasher(doConfigure, command)
+	b.start()
+}
+
+func doConfigure(success chan bool, command string) {
+	posh := powershell.New()
+	result, _ := posh.Execute(command)
+
+	if strings.Contains(result, "4096") {
```

Comment:
What is `4096` here?

Reply:
It is success result code for WMI objects. Can't help the weird numbering... But this is the implementation.

Reply:
Since it is followed by
```
  success <- true
```
I guess it is/was clear enough